### PR TITLE
Updated PV names provided by IIB boards (DEPRECATED!)

### PR DESCRIPTION
**Pull request by gabrielbrunheira**
This is a suggestion for a new standard of PV names from variables provided by IIB boards from FAP and FAC power supplies. I believe this should be extended for Archiver, right?

### CODE REVIEW

File: siriuspy/siriuspy/pwrsupply/csdev.py [hunk 1]
```diff
@@ -1334,13 +1333,13 @@ def _get_ps_FAP_propty_database():
...
-                                       'prec': 2,
-                                       'unit': 'C'},
-        'IIBHeatSinkTemperature-Mon': {'type': 'float', 'value': 0.0,
+        'InductorsTemperatureIIB-Mon': {'type': 'float', 'value': 0.0,
+                                        'prec': 2,
+                                        'unit': 'C'},
+        'HeatSinkTemperatureIIB-Mon': {'type': 'float', 'value': 0.0,
                                        'prec': 2,
                                        'unit': 'C'},
         'DCLinkVoltage-Mon': {'type': 'float', 'value': 0.0,
```
Comment (anacso17) [inline siriuspy/siriuspy/pwrsupply/csdev.py:1345]: Following the new standard, shouldn't it be _DCLinkVoltageIIB-Mon?_

Comment (gabrielbrunheira) [reply siriuspy/siriuspy/pwrsupply/csdev.py:1345]: No, because this specific variable isn't provided b y IIB in this PS model. VoltageInputIIB is indeed.

**Comment by xresende** [comment]
> @anacso17, @gabrielbrunheira, It seems there is something wrong with this PR. there is a lot of news from the master branch here. But it seems they aren't in the branch you guys want to merge with.
> I think this branch must be closed and re-branched from the base branch you want to merge. Then I think you should cherry-pick the commits you want to add.
> Don't you guys think?

I merged master to the base branch of this PR and the merged master to its the working branch but the PR should many commits (already merged) as new commit belonging to this PR. git diff shows proper difference! I don't understand why the PR shows all these commit already merged!

**Comment by anacso17** [comment]
@xresende it seems that the remote base branch `PR-update-ioc-for-firmware-V1.25-0.42-FBP-FAP` is not synchronized with master, but this one is.

**Comment by xresende** [comment]
> @xresende it seems that the remote base branch `PR-update-ioc-for-firmware-V1.25-0.42-FBP-FAP` is not synchronized with master, but this one is.

it is! I just did this right before updating `update-ps-pvnames`!

**Comment by fernandohds564** [comment]
> > @anacso17, @gabrielbrunheira, It seems there is something wrong with this PR. there is a lot of news from the master branch here. But it seems they aren't in the branch you guys want to merge with.
> > I think this branch must be closed and re-branched from the base branch you want to merge. Then I think you should cherry-pick the commits you want to add.
> > Don't you guys think?
> 
> I merged master to the base branch (`PR-update-ioc-for-firmware-V1.25-0.42-FBP-FAP`) of this PR and then merged master to its the working branch (`update-ps-pvnames`) but the PR shows many commits (already merged) as new commits belonging to this PR. git diff shows proper difference! I don't understand why the PR shows all these commit already merged!
> 
> Maybe @gabrielbrunheira could make a patch from `git diff PR-update-ioc-for-firmware-V1.25-0.42-FBP-FAP > patch.txt` and recreate the PR.

I don't understand why this happens either. However, I think the most appropriate way of doing that would be to merge the master branch to the base branch and then the base branch into this branch. Maybe when you merged master into both branches, the commits hashes were recreated twice?!?

I think we should start using rebase...
But for now, maybe your idea of recreating the PR through `git diff` is good!

**Comment by xresende** [comment]
> > > @anacso17, @gabrielbrunheira, It seems there is something wrong with this PR. there is a lot of news from the master branch here. But it seems they aren't in the branch you guys want to merge with.
> > > I think this branch must be closed and re-branched from the base branch you want to merge. Then I think you should cherry-pick the commits you want to add.
> > > Don't you guys think?
> > 
> > 
> > I merged master to the base branch (`PR-update-ioc-for-firmware-V1.25-0.42-FBP-FAP`) of this PR and then merged master to its the working branch (`update-ps-pvnames`) but the PR shows many commits (already merged) as new commits belonging to this PR. git diff shows proper difference! I don't understand why the PR shows all these commit already merged!
> > Maybe @gabrielbrunheira could make a patch from `git diff PR-update-ioc-for-firmware-V1.25-0.42-FBP-FAP > patch.txt` and recreate the PR.
> 
> I don't understand why this happens either. However, I think the most appropriate way of doing that would be to merge the master branch to the base branch and then the base branch into this branch. Maybe when you merged master into both branches, the commits hashes were recreated twice?!?
> 
> I think we should start using rebase...
> But for now, maybe your idea of recreating the PR through `git diff` is good!

I usually do what you proposed: merge master into the base branch and then merge the base branch into the PR branch. But at the end it should be equivalent to what I dod this time, as far as I understand.

**Comment by xresende** [comment]
- I removed @fernandohds564 as reviewer because I thought he had been included automaticaly after I merged master into either the base branch or the PR branch. I only saw you did it, @gabrielbrunheira , minutes ago.
- @gabrielbrunheira , I will send you the patch file you can use to recreate you PR over `PR-update-ioc-for-firmware-V1.25-0.42-FBP-FAP` with:

1. git checkout PR-update-ioc-for-firmware-V1.25-0.42-FBP-FAP
2. git apply update-ps-pvnames.patch

**Comment by gabrielbrunheira** [comment]
This PR will closed and replaced by PR https://github.com/lnls-sirius/dev-packages/pull/551

**Comment by gabrielbrunheira** [comment]
This PR will closed and replaced by PR #551

**Pull request closed without merging**